commands/pull.go

```diff
@@ -67,7 +67,7 @@ func (c *singleCheckout) Run(p *lfs.WrappedPointer) {
 	// Check the content - either missing or still this pointer (not exist is ok)
 	filepointer, err := lfs.DecodePointerFromFile(cwdfilepath)
 	if err != nil && !os.IsNotExist(err) {
-		if errors.IsNotAPointerError(err) || errors.IsBadPointerKeyError(err) {
+		if errors.IsNotAPointerError(err) {
```

Comment:
Was scratching my head for a while, until I found this line of code, which was silencing the error "expected key oid, got <<<<<".

This line seems to have been added fairly recently, in v2.13.2 ([commit](https://github.com/git-lfs/git-lfs/commit/9e3bc45b934758e422c3e800161dabd0711775bf))

My git installation was still runnign git-lfs 2.11.0, which explains why I could repro easily using my regular git, but not with the latest version.  

Is silencing the error still the expected behaviour? I find that this sometimes leads to accidently commiting files with conflics (e.g. _BACKUP files). While it's not a terrible problem, it seems undesirable.

Reply:
If you want to revert it, that's fine, but please remove the test so it doesn't fail.

Reply:
It's probably good 'as is'.